x-pack/solutions/security/packages/ecs-data-quality-dashboard/impl/data_quality_panel/mock/test_providers/test_providers.tsx

```diff
@@ -83,6 +84,17 @@ const TestExternalProvidersComponent: React.FC<TestExternalProvidersProps> = ({
               basePath={'https://localhost:5601/kbn'}
               docLinks={{
                 ELASTIC_WEBSITE_URL: 'https://www.elastic.co/',
+                links: {
+                  securitySolution: {
+                    elasticAiFeatures: `${ELASTIC_DOCS}solutions/security/ai`,
+                    thirdPartyLlmProviders: `${ELASTIC_DOCS}solutions/security/ai/set-up-connectors-for-large-language-models-llm`,
+                    llmPerformanceMatrix: `${ELASTIC_DOCS}solutions/security/ai/large-language-model-performance-matrix`,
+                  },
+                  alerting: {
+                    elasticManagedLlm: `${ELASTIC_DOCS}reference/kibana/connectors-kibana/elastic-managed-llm`,
+                    elasticManagedLlmUsageCost: `https://www.elastic.co/pricing`,
+                  },
+                } as DocLinksStart['links'],
                 DOC_LINK_VERSION: 'current',
               }}
```

Comment:
This could cause problems since we are only defining a small set of links, and they are cast to the entire type. This would be acceptable inside a specific test, but it is a bit dangerous in reusable tools like this one.

Consider mocking it properly:

```suggestion
              docLinks={docLinksServiceMock.createStartContract()}
```

The same with the rest of test provider mocks.